citi-onboarding/acervusej-onepage
# Fix/gallery

**Things I've done:**

- Adjusted the slick track and slick list css

**How to test it:**

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write http://localhost:3000/.
- See if everything works in gallery.
